File: ejercicios-js/ejercicio5.js
const secreto = Math.floor(Math.random() * 100) + 1;
let intento;
do {
  intento = Math.floor(Math.random() * 100) + 1;
  if (intento < secreto) {
    console.log(`${intento} es menor ❌`);
  } else if (intento > secreto) {
    console.log(`${intento} es mayor ❌`);
  }
} while (intento !== secreto);
console.log(`¡Correcto! El número era ${secreto} ✅`);

File: ejercicios-py/ejercicio5.py
import random

secreto = random.randint(1, 100)
while True:
    intento = random.randint(1, 100)
    if intento < secreto:
        print(f"{intento} es menor ❌")
    elif intento > secreto:
        print(f"{intento} es mayor ❌")
    else:
        break
print(f"¡Correcto! El número era {secreto} ✅")